src/App.tsx

```diff
@@ -15,48 +16,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[], sortType: SortType, isReversed: boolean,
+) {
+  // To avoid the original array mutation
```

Comment:
It's better to write code that doesn't require comments to be understandable. In this case it'd be good even without comment so just remove it.